### DISCUSSION TITLE
Can you please exaplain to me easily the difference between `fallback` and `fallback_service`?

### DISCUSSION THREAD

**Discussion by frederikhors** [tokio-rs/axum#2500] [Q&A] [answered]
### Summary

Can you please exaplain to me easily the difference between `fallback` and `fallback_service`?

- https://docs.rs/axum/latest/axum/routing/struct.Router.html#method.fallback

- https://docs.rs/axum/latest/axum/routing/struct.Router.html#method.fallback_service

### axum version

0.6 or 0.7

**Comment by davidpdrsn** [comment, marked as answer]
Dunno what else to say besides whats in the docs. `fallback` takes a handler, `fallback_service` takes a service.

**Reply by frederikhors** [reply]
Sorry, my mistake.
